rlp/from.go

```diff
@@ -31,14 +31,25 @@ func from(input string) (*Value, string, error) {
 	// And of course the RLP wiki spec:
 	//   https://github.com/ethereum/wiki/wiki/RLP
 
+	if input == "" {
+		return &Value{String: "0x"}, "", nil
+	}
+
+	for _, r := range input {
+		if ('a' > r || r > 'f') && ('A' > r || r > 'F') && ('0' > r || r > '9') {
```

Comment:
```suggestion
		if (r < 'a' || 'f' < r) && (r < 'A' || 'F' < r) && (r < '0' || '9' < r) {
```

Reply:
Maybe it's just me but it took me way too long to understand this condition. The way it's written make me think about `r` being within those interval where it's in fact the opposite. Feel free to take the suggestion or not.